WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -256,11 +256,23 @@ struct CouponDetails: View {
                 Text(String.localizedStringWithFormat(Localization.maximumSpend, viewModel.maximumAmount))
                     .renderedIf(viewModel.maximumAmount.isNotEmpty)
 
+                Text(String.localizedStringWithFormat(Localization.singularUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit == 1)
+
+                Text(String.localizedStringWithFormat(Localization.pluralUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit > 1)
+
```

Comment:
Suggestion: we can simplify this using the `pluralize` extension method for `String`. The only problem is that we're supporting `Int` amount only, should we add a new method to support `Int64`? With that we can make it simpler like so:

```suggestion
                Text(String.pluralize(viewModel.usageLimit,
                                      singular: Localization.singularUsageLimitPerCoupon,
                                      plural: Localization.pluralUsageLimitPerCoupon))
                    .renderedIf(viewModel.usageLimit > 0)
```

WDYT?

Reply:
Good one! I've fixed it in both these commits: https://github.com/woocommerce/woocommerce-ios/pull/6831/commits/1e0a4060d199e0ece9bf9353347bdd4b02ff485d and https://github.com/woocommerce/woocommerce-ios/pull/6831/commits/a312d0fb86d7fb9ccdfed2bb78210c557e908766.

But I have one newbie question: Do we still support 32bit devices in our iOS versions? If not, wouldn't every `Int` already be by default an `Int64`? Or Swift forces every `Int` to be `Int32`? I ask this because I was thinking if these properties shouldn't be converted directly to `Int`, this way we could drop the pluralize I added for `Int64` and only use the existing one. 🤔

Reply:
Great question! 💯 It is true that we can actually cast `Int64` to `Int` since Apple dropped support for 32bit systems and forced all apps to support 64bit since iOS 11. I think we're still using `Int64` to explicitly state the support for IDs and values that can be quite large. This also keeps it consistent with the practice we've been doing since the "old days".

With that said, I believe can drop the `Int64` variant of the `pluralize` method - thank you for pointing that out!